test/tizen_capi/unittest_tizen_capi_dataset.cpp

```diff
@@ -566,6 +566,64 @@ TEST(nntrainer_capi_dataset, set_dataset_property_for_mode_02_n) {
   EXPECT_EQ(status, ML_ERROR_NONE);
 }
 
+/**
+ * @brief Neural Network Dataset set Property Test (positive test)
+ */
+TEST(nntrainer_capi_dataset,
+     set_dataset_property_for_mode_with_single_param_03_p) {
+  ml_train_dataset_h dataset;
+  int status = ML_ERROR_NONE;
+
+  status = ml_train_dataset_create(&dataset);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  status = ml_train_dataset_set_property_for_mode_with_single_param(
+    dataset, ML_TRAIN_DATASET_MODE_TRAIN, "buffer_size=1");
+  EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
```

Comment:
I think this does not need here because it will be tested in `set_dataset_property_for_mode_with_single_param_04_n`